### PR TITLE
Fixed extra scroll-bar bugs && changed accent color.

### PR DESCRIPTION
**Pull request by anuragts**
##### index page
image

##### index page
image

##### index page
image
##### index page
image

### CODE REVIEW

File: src/pages/signup.jsx
Comment (anuragts) [inline src/pages/signup.jsx:1]: previously there was no red color in error message

File: tailwind.config.cjs [hunk 1]
```diff
@@ -16,7 +16,7 @@ module.exports = {
     extend: {
       colors: {
         brand: "#242424",
-        accent: "#B73535",
```
Comment (anuragts) [inline tailwind.config.cjs:19]: Added a bright red accent (looks bit more good in dark mode)

File: src/pages/home.jsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ import BrandLogo from "../assets/images/icon.png";
 const Home = (props) => {
   return (
     <main className="bg-brand font-inter h-screen w-screen inset-0 text-white flex flex-col overflow-y-scroll overflow-x-hidden justify-center items-center">
-      <div className="bg-brand font-inter py-6 px-6 2xl:px-[600px] xl:px-[400px] lg:px-[300px] flex flex-col overflow-y-scroll justify-center items-center">
```
Comment (anuragts) [inline src/pages/home.jsx:16]: idk why u added this ????

**Comment by aminoxix** [comment]
@anuragts Thanks for your contribution, looking forward to reviewing! 🙂

File: src/pages/signup.jsx [hunk 3]
```diff
@@ -60,7 +60,7 @@ const SignUp = () => {
       !/[A-Za-z0-9!@#$%^&*()_+]/.test(values.rawPassword) ||
       values.rawPassword.length <= 8
     ) {
-      setErrorMessage("Please fill the details carefully");
+      setErrorMessage( <div className="text-red-500 font-semibold text-sm">Please fill the details carefully</div>);
```
Comment (aminoxix) [inline src/pages/signup.jsx]:
```suggestion
      setErrorMessage("Please fill the details carefully");
```

1. please check the indentation of this line using a prettier formatter.
2. check the line [line #231](https://github.com/aminoxix/BoilerplateWebApp/blob/main/src/pages/signup.jsx#L231) & do the necessary changes!

**Review by aminoxix — APPROVED** [review]
✅  LGTM!

**Pull request merged**